python/demo/gsoc2022/gmsh_helpers.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (C) 2020 Jørgen S. Dokken
```

Comment:
Is this built in to DOLFINx now?

Reply:
No, it is not. We could remove it (or move it into DOLFINx.io.gmsh). It just gets very tedious to generate meshes and use them without such a helper function.

Especially as we do not want to save mesh files in the git repo.

Reply:
@jorgensd  Could you make a PR with that?
Otherwise the demo becomes too long and difficult to follow.

Reply:
@garth-wells  What do you think? I think it would be very neat to have a way of accessing gmsh meshes without having to go through meshio. It would otherwise be quite tricky to have "advanced" demos, as mesh generation and conversion would take up most of the demo. Especially if I update: https://github.com/FEniCS/dolfinx/pull/2131 such that gmsh is a sub-module, this should not be intrusive.